sdk/eventhub/azure-eventhub/azure/eventhub/_producer.py

```diff
@@ -136,6 +136,8 @@ def __init__(
     def _create_handler(
         self, auth: Union[uamqp_JWTTokenAuth, JWTTokenAuth]
     ) -> None:
+
+        desired_capabilities = [GEOREPLICATION_SYMBOL]
```

Comment:
today this is a no-op?

Reply:
yeah, no desired capabilities on the send side